### PR TITLE
"http://www.pornhub.com Station 13" is automatically approvable.

### PR DESCRIPTION
**Pull request by Iamgoofball**
This fixes that.

### CODE REVIEW

**Comment by MrStonedOne** [comment]
\>exploit
\>not putting [s]

**Comment by Iamgoofball** [comment]
fuck

File: code/game/objects/items/weapons/charter.dm [hunk 1]
```diff
@@ -51,11 +51,6 @@
 	log_game("[key_name(user)] has proposed to name the station as \
 		[new_name]")
 
-	if(standard_station_regex.Find(new_name))
```
Comment (Giacom) [inline code/game/objects/items/weapons/charter.dm]: Is standard_station_regex only used here? If so, you could remove it and any related code.

Comment (RemieRichards) [inline code/game/objects/items/weapons/charter.dm]: this is a shitpost pr gia, check out the one linked above.

Comment (Giacom) [inline code/game/objects/items/weapons/charter.dm]: Oh, well then

**Comment by Cyberboss** [comment]
Going with the other PR

File: code/game/objects/items/weapons/charter.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/game/objects/items/weapons/charter.dm]: yeah other one is better

**Pull request closed without merging**